src/universal/utils/promiseAllPartial.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Promise.all fails fast, which means if promsie 2 of 100 fails, it aborts on 2
+ * Sometimes, we want all the results & replace the errors with some generic value like null
+ * This does that, and does it quickly, assuming that the first promise in the array will resolve first
+ */
+
+const promiseAllPartial = async (promiseArr, catchValue = null) => {
```

Comment:
I am shocked that this isn't an option to Promise.all, right??

Reply:
not only that, but it's not an option for the `for-await-of`either (http://2ality.com/2017/12/for-await-of-sync-iterables.html). I mean, if i use Promise.all, i want to execute in parallel. if i execute in parallel, why the heck would i want serial pass/fail dependency? TC39 is weird.